### DISCUSSION TITLE
Panel Link on Stat

### DISCUSSION THREAD

**Discussion by Mueke666** [grafana/grafana#39395] [Dashboards, Panels & Visualizations]
Hello Everybody,

can you implement the Feature, that the Panel Link will open when i click on the Panel it self and not on the little Button.
For Touch Devices ist the little Button not nice.

Greetings from Germany

**Comment by torkelo** [comment]
if you use a data link you can click on the whole panel

**Reply by Mueke666** [reply]
Hey, this is exactly that what i mean. 
Thank You!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
